.github/workflows/unit-integration-test.yml

```
@@ -10,7 +10,7 @@ jobs:
         strategy:
             fail-fast: false
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)